changeFeedSamples/trackingChangesToBlobs/src/main/java/ChangeFeedTimer.java

```diff
@@ -0,0 +1,201 @@
+        this.createAdditionalEvents = createAdditionalEvents;
+        this.serviceClient = blobServiceClient;
+        this.containerClient = containerClient;
+        this.blobClient = blobClient;
+        this.changefeedClient = changefeedClient;
+        this.cursor = cursor;
+    }
+
+    /**
+     * Every time the timer executes, check for new events starting from where cursor points at and filter events. Filter
+     * depends on how the predicates are set up.
+     */
+    public void run() {
+        BlobChangefeedPagedIterable iterable = null;
+
+        // Starting from cursor position if there exists one
+        if (this.cursor == null) {
+            iterable = this.changefeedClient.getEvents();
+        }
+        else {
+            iterable = this.changefeedClient.getEvents(this.cursor);
+        }
+
+        Stream<BlobChangefeedPagedResponse> pages = iterable.streamByPage(1);
+
+        System.out.println("Printing all events satisfying filters");
+
+        // Checking by page every event and seeing if it satisfies filters. At the end, store the cursor in a blob
+        pages.forEach(page -> {
+                    page.getElements().stream()
+                            .filter(event ->
+                                    // TODO: Change predicate chain to desired chain to filter events
+                                    checkBlobName.and(checkContainerName).test(event))
+                            .forEach(event ->
+                                    // TODO: Change output as needed to see necessary information from event
+                                    System.out.printf("Time: %s, Subject: %s, ID: %s, Type: %s%n",
+                                            event.getEventTime(), event.getSubject(), event.getId(), event.getEventType()));
+                    this.cursor = page.getContinuationToken();
+                }
+        );
+
+        System.out.println("Printed all events satisfying filter since last check, storing cursor into storage account");
+        // Stores cursor in storage account, in case if it needs to be used again later
+        this.storeCursor();
+        System.out.println("Stored cursor");
+        if (this.createAdditionalEvents) {
+            this.addMoreEvents();
+        }
+    }
+
+    /**
+     * Stores cursor into blob in a container
+     */
+    private void storeCursor() {
+        // Creates container if there is no container
+        if (!this.containerClient.exists()){
+            this.containerClient.create();
+        }
+
+        // Storing cursor in blob
+        BlockBlobClient blockBlobClient = this.blobClient.getBlockBlobClient();
+        ByteArrayInputStream dataStream = new ByteArrayInputStream(this.cursor.getBytes());
+        blockBlobClient.upload(dataStream, this.cursor.length(), true);
+    }
+
+    /**
+     * Creates additional events (optional if changefeed is to be run for a long time)
+     */
+    private void addMoreEvents() {
```

Comment:
Do we want this to be part of the Changefeed sample or part of setup code that they could run on the side?